### DISCUSSION TITLE
testcontainers/sshd source codes

### DISCUSSION THREAD

**Discussion by paveljandejsek** [testcontainers/testcontainers-java#4268] [Q&A] [answered]
Hi everyone, apologies if I overlooked it but can I please ask where the dockerfile for `testcontainers/sshd` is located?

I found it on dockerhub both [here](https://hub.docker.com/r/testcontainers/sshd) and [here](https://hub.docker.com/r/testcontainersofficial/sshd), but both of those link to the same [repo](https://www.github.com/testcontainers/sshd-docker) that does not seem to exist and gives me 404...
Or is there some documentation for it somewhere?

Or is this old/not used anymore? (from the docs I presume that it is being used when exposing host ports to containers, but I havent been able to find much else)

Thanks in advance

**Comment by bsideup** [comment, marked as answer]
Hi @paveljandejsek,

The repo was private by accident.

I just changed it to "public", so that you can verify its source:
<img width="665" alt="Screenshot 2021-07-07 at 08 04 57" src="https://user-images.githubusercontent.com/1050762/124708113-13c80d00-defa-11eb-825d-51b8674725c1.png">

Thanks for telling us about it! 💯

**Reply by paveljandejsek** [reply]
Thanks a lot, much appreciated!
